<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PokeEdit</class>
 <widget class="QWidget" name="PokeEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,100,0,0,0,0">
   <item row="0" column="3" colspan="3">
    <widget class="QPushButton" name="done">
     <property name="text">
      <string>&amp;Trainer Home</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="pokemonFrame">
     <property name="minimumSize">
      <size>
       <width>96</width>
       <height>96</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>96</width>
       <height>96</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="itemSprite">
      <property name="geometry">
       <rect>
        <x>72</x>
        <y>72</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="pokemonSprite">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>96</width>
        <height>96</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="speciesLabel">
         <property name="text">
          <string>Species</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="genderSprite">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="type1">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="type2">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="nickname">
       <property name="placeholderText">
        <string>Pokemon nickname</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Nature</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="nature"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Item</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="item"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Happiness</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="happiness">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="2" colspan="4">
    <widget class="EvBox" name="evbox" native="true"/>
   </item>
   <item row="2" column="5" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QGroupBox" name="finalStats">
       <property name="title">
        <string>Final Stats</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="verticalSpacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="hpivdesc">
          <property name="text">
           <string>HP:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="hpivlabel">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="attivdesc">
          <property name="text">
           <string>Att:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="attivlabel">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="defivdesc">
          <property name="text">
           <string>Def:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="defivlabel">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="spattivdesc">
          <property name="text">
           <string>Sp. Att:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="spattivlabel">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="spdefivdesc">
          <property name="text">
           <string>Sp. Def:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="spdefivlabel">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="speedivdesc">
          <property name="text">
           <string>Speed:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="speedivlabel">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="hpivspin">
          <property name="maximum">
           <number>31</number>
          </property>
          <property name="value">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="attivspin">
          <property name="maximum">
           <number>31</number>
          </property>
          <property name="value">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="defivspin">
          <property name="maximum">
           <number>31</number>
          </property>
          <property name="value">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QSpinBox" name="spattivspin">
          <property name="maximum">
           <number>31</number>
          </property>
          <property name="value">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QSpinBox" name="spdefivspin">
          <property name="maximum">
           <number>31</number>
          </property>
          <property name="value">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QSpinBox" name="speedivspin">
          <property name="maximum">
           <number>31</number>
          </property>
          <property name="value">
           <number>31</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="hiddenPowerTitle">
         <property name="text">
          <string>Hidden Power</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="hiddenPowerType"/>
       </item>
       <item>
        <widget class="QLabel" name="hiddenPowerPower">
         <property name="text">
          <string>Power: 70</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="abilityTitle">
         <property name="text">
          <string>Ability</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="ability1">
         <property name="text">
          <string>Ability 1</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="ability2">
         <property name="text">
          <string>Ability 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="ability3">
         <property name="text">
          <string>DW Ability</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="6" rowspan="2">
    <widget class="PokeLevelSettings" name="levelSettings"/>
   </item>
   <item row="2" column="0" colspan="5">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0" colspan="4">
      <widget class="QTableView" name="moveChoice">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>22</number>
       </attribute>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="move1">
       <property name="placeholderText">
        <string>Move 1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="move2">
       <property name="placeholderText">
        <string>Move 2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="move3">
       <property name="placeholderText">
        <string>Move 3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="move4">
       <property name="placeholderText">
        <string>Move 4</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EvBox</class>
   <extends>QWidget</extends>
   <header>evbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PokeLevelSettings</class>
   <extends>QFrame</extends>
   <header>pokelevelsettings.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>move1</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>move1</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>move2</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>move2</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>move3</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>move3</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>526</x>
     <y>447</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>move4</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>move4</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>648</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>744</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
